src/web/docusaurus/docs/api-services/posts.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 3
+---
+
+# Posts Service
+
+## Overview
+
+The posts service is responsible for retrieving posts that are cached in redis.
+
+## API endpoints
+
+### Get posts
+
+Find any posts that are currently cached in redis.
+
+```
+GET /
+```
+
+#### Code Samples
+
+##### Shell
+
+```bash
+curl -X GET \
+  http://localhost/v1/posts/
+```
+
+##### JavaScript
+
+- With `fetch`
+
+```js
+fetch('http://localhost/v1/posts/');
+```
+
+#### Responses
+
+##### Successful response
+
+```
+Status: 200 OK
+```
+
+```json
+{
+  "posts": [
+    { "id": "64f3152a00", "url": "https://dev.api.telescope.cdot.systems/v1/posts/64f3152a00" }
+  ]
+}
+```
+
+### Get a single post
+
+Find a single post that is currently cached in redis. Response can be retrieved as html, json, or plain-text. Specify as part of the headers what you want the response to look like.
+
+```
+GET /:id
+```
+
+#### Parameters
+
+| Name | Type   | In    | Description                         |
+| ---- | ------ | ----- | ----------------------------------- |
+| id   | string | query | _Required_: The id of the blog post |
+
+#### Code Samples
+
+##### Shell
+
+```bash
+curl -X GET \
+  http://localhost/v1/posts/64f3152a00
+```
+
+##### JavaScript
+
+- With `fetch`
+
+```js
+fetch('http://localhost/v1/posts/64f3152a00');
+```
+
+#### Responses
+
+##### Successful response
+
+```
+Status: 200 OK
+```
+
+```json
+{
+  id: '28867e65b0',
```

Comment:
I'm not sure why this is in red highlight? Is that just my computer?

Reply:
I forgot to put quotes around the key name, it's red cause it's not valid in json, gonna fix that